config/500-dispatcher.yaml

```diff
@@ -50,6 +50,10 @@ spec:
               value: config-logging
             - name: METRICS_DOMAIN
               value: knative.dev/eventing
+            - name: DEFAULT_NATSS_URL
+              value: nats://nats-streaming.natss.svc.cluster.local:4222
```

Comment:
Tricky bit about this is that for clusters that have different extensions (not cluster.local), this will not work at all. But, it's a default :) Just wanted to point it out explicitly.

Reply:
yeah,just wanted to point it out explicitly.